### PR TITLE
Chore/remove oz test helpers and web3

### PR DESCRIPTION
**Pull request by MilGard91**
## Context

Issue: #440 

## Changes proposed in this pull request

Cleaning up the the tests by removing oz test helpers and web3.js from them.

## Test plan

N/A

## Checklist

- [x] Rebased the base branch
- [x] Attached corresponding Github issue
- [x] Prefixed the name with the type of change (i.e. feat, chore, test)
- [x] Performed a self-review of my own code
- [x] Followed the style guidelines of this project
- [ ] Made corresponding changes to the documentation
- [x] Didn't generate new warnings
- [x] Didn't generate failures on existing tests
- [ ] Added tests that prove my fix is effective or that my feature works

## Review

When reviewing a PR, please indicate intention in comments using the following emojis:
* :cake: = Nice to have but not essential.
* :bulb: = Suggestion or a comment based on personal opinion
* :hammer: = I believe this should be changed.
* :thinking: = I don’t understand something, do you mind giving me more context?
* :rocket: = Feedback

### CODE REVIEW

File: test/unit/NXMaster/addNewInternalContracts.js [hunk 1]
```diff
@@ -14,38 +13,35 @@ describe('addNewInternalContracts', function () {
...
+    expect(
+      governance.addNewInternalContracts([hex('XX')], [AddressZero], [ContractTypes.Replaceable]),
+    ).to.be.revertedWith('NXMaster: Contract address is 0');
   });
 
   it('reverts when contract type is unknown', async function () {
     const { governance } = this;
 
-    await expectRevert(
+    expect(
```
Comment (danoctavian) [inline test/unit/NXMaster/addNewInternalContracts.js:42]: same here 

await expect

File: test/unit/NXMaster/addNewInternalContracts.js [hunk 2]
```diff
@@ -14,38 +13,35 @@ describe('addNewInternalContracts', function () {
   it('reverts when not called by governance', async function () {
     const { master } = this;
 
-    await expectRevert(master.addNewInternalContracts([], [], []), 'Not authorized');
+    expect(master.addNewInternalContracts([], [], [])).to.be.revertedWith('Not authorized');
```
Comment (danoctavian) [inline test/unit/NXMaster/addNewInternalContracts.js:16]: await expect

File: test/unit/NXMaster/addNewInternalContracts.js [hunk 3]
```diff
@@ -14,38 +13,35 @@ describe('addNewInternalContracts', function () {
...
   });
 
   it('reverts when contract code already in use', async function () {
     const { governance } = this;
 
-    await expectRevert(
+    expect(
```
Comment (danoctavian) [inline test/unit/NXMaster/addNewInternalContracts.js:22]: await expect

File: test/unit/NXMaster/removeContracts.js [hunk 4]
```diff
@@ -13,13 +12,13 @@ describe('removeContracts', function () {
   it('reverts when not called by governance', async function () {
     const { master } = this;
 
-    await expectRevert(master.removeContracts([]), 'Not authorized');
+    expect(master.removeContracts([])).to.be.revertedWith('Not authorized');
```
Comment (danoctavian) [inline test/unit/NXMaster/removeContracts.js:15]: await expect

File: test/unit/NXMaster/upgradeMultipleContracts.js [hunk 5]
```diff
@@ -1,30 +1,27 @@
...
-    );
+    ).to.be.revertedWith('NXMaster: Non-existant or non-upgradeable contract code');
   });
 
   it('reverts when contract address is 0t', async function () {
     const { governance } = this;
 
-    await expectRevert(
-      governance.upgradeMultipleContracts([hex('GV')], [ZERO_ADDRESS]),
+    expect(governance.upgradeMultipleContracts([hex('GV')], [AddressZero])).to.be.revertedWith(
```
Comment (danoctavian) [inline test/unit/NXMaster/upgradeMultipleContracts.js:24]: await expect

File: test/utils/governance.js [hunk 6]
```diff
@@ -38,14 +37,14 @@ const submitMemberVoteProposal = async (gv, pc, categoryId, actionData, members)
...
+  await increaseTime(closingTime.addn(1).toString());
   await gv.closeProposal(proposalId, { from: members[0] });
 
   const { val: speedBumpHours } = await gv.getUintParameters(hex('ACWT'));
-  await time.increase(speedBumpHours.muln(3600).addn(1).toString());
+  await increaseTime(speedBumpHours.muln(3600).addn(1).toString());
   const triggerTx = await gv.triggerAction(proposalId);
 
-  expectEvent(triggerTx, 'ActionSuccess', { proposalId });
+  await expect(triggerTx).to.emit(gv, 'ActionSuccess').withArgs(proposalId);
```
Comment (danoctavian) [inline test/utils/governance.js:47]: is the await here redundant bc it was already awaited? i think so

**Review by danoctavian — APPROVED** [review]
LGTM

There may be some redundancy with the await in some cases, where the tx is awaited and then the expect is awaited again

**Review by roxdanila — APPROVED** [review]

**Pull request merged**